lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_git_helper.rb

```diff
@@ -14,7 +14,7 @@ module GitHelper
         def self.commit_version_bump
           Fastlane::Helper::GitHelper.commit(
             message: 'Bump version number',
-            files: File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'], 'build.gradle'),
+            files: File.join(ENV['PROJECT_ROOT_FOLDER'], 'version.properties'),
```

Comment:
This suggests that all host/app projects migrate to this new `version.properties` convention.

I'm all for this convention and separate `version.properties` file, seems cleaner 👍 but that also means that this will be a breaking change that will require SNAndroid, WCAndroid and WPAndroid to be updated accordingly before they can adopt the new toolkit.

I suggest that we:
 - Either make this change so that it's compatible with both versions, to ease the transition from the old way to the new way. Maybe with a flag to pass to the actions to do one or the other. Drawback of this is that it will complexify the code and likely be the kind of "patch" that will stay for a while before we clean things up, so maybe not the ideal path
 - Or create new actions, which use the new way, and deprecate the old actions
 - And/Or wait for us to do our first stable 1.0 – to which all repos can point to, then immediately prepare for a 2.0 – with those breaking changes in – with semver telling us that this will be a breaking change and will require all host app / clients to update their fastfiles in parallel of updating the release-toolkit
 - Lastly we could do one last `0.x.y` version before that one, and make 1.0 be that the version that holds the breaking change. There are still a lot couple of non-breaking changes pending PR approval that I'd want us to release first before introducing new breaking changes and a new major version bump tho, so that apps that need the pending fixes (GH releases fixes mainly) but are not yet ready to migrate to this new `version.properties` on Android can still have a release to point to.